libcasr/fuzz/fuzz_targets/parse_stacktrace.rs

```diff
@@ -0,0 +1,49 @@
+#![no_main]
+
+use libfuzzer_sys::fuzz_target;
+
+use libcasr::asan::AsanStacktrace;
+use libcasr::gdb::GdbStacktrace;
+use libcasr::go::GoStacktrace;
+use libcasr::java::JavaStacktrace;
+use libcasr::python::PythonStacktrace;
+use libcasr::stacktrace::ParseStacktrace;
+
+fuzz_target!(|data: &[u8]| {
+    if data.len() < 2 {
+        return;
+    }
+    let s = String::from_utf8_lossy(&data[1..]);
+    match data[0] % 5 {
+        0 => {
+            // Asan
+            if let Ok(st) = AsanStacktrace::extract_stacktrace(&s) {
+                let _ = AsanStacktrace::parse_stacktrace(&st);
```

Comment:
May be we need to add `crash_line()` func using?

Reply:
Yeah, that would be nice